src/proto/console/pointer/mod.rs

```diff
@@ -63,9 +63,9 @@ impl<'boot> Pointer<'boot> {
 pub struct PointerMode {
     // The pointer device's resolution on the X/Y/Z axis in counts/mm.
     // If a value is 0, then the device does _not_ support that axis.
```

Comment:
Could you change these normal comments to doc comments (`///`) so that they'll show up as documentation for this field?

Reply:
Sure!

Reply:
Hmm, it seems the fix was lost in a rebase :thinking:?